### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by simbamford**
Added two software packages: bimvee, and mustard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1848,6 +1848,8 @@ arXiv, 2020.
 - Marcireau A., Ieng S.-H., Benosman R.,  
 *[Sepia, Tarsier, and Chameleon: A Modular C++ Framework for Event-Based Computer Vision](https://doi.org/10.3389/fnins.2019.01338)*,  
 Front. Neurosci. (2020), 13:1338. [Code](https://github.com/neuromorphic-paris/tutorials)
+- [BIMVEE](https://github.com/event-driven-robotics/bimvee) Python tools for Batch Import, Manipulation, Visualisation and Export of Events and other timestamped data. Imports from various file formats into a common workspace format, including native Python import of rosbags.
+- [MUSTARD](https://github.com/event-driven-robotics/mustard) Python app for multistream synchronised playback of asynchronous data, including events, frames, pose, labels, optic flow etc.
```
Comment (guillermogb) [inline README.md:1852]: I could not access this repo. Is it publicly available?

Comment (guillermogb) [reply README.md:1852]: (Mustard)

Comment (simbamford) [reply README.md]: Sorry - Indeedd that repo hasn't been made public yet - I've changed the PR to remove that.

Comment (guillermogb) [reply README.md]: no worries

**Comment by guillermogb** [comment]
Looking forward to seeing Mustard! :P

**Pull request merged**